### PR TITLE
Tagesanzeiger.ch:

### PR DESCRIPTION
**Pull request by numismatika**
- add more subdomains

### CODE REVIEW

**Comment by J0WI** [comment]
I think this host is similar to #8548: They just use a wildcard DNS and sub domains for some top stories.

**Comment by numismatika** [comment]
It id the same media holding so very likely.

**Comment by J0WI** [comment]
Could you add a hint about this behaviour in the comments?

**Comment by J0WI** [comment]
@numismatika I agree with this. But now you could also add `foo.tagesanzeiger.ch` and `bar.tagesanzeiger.ch` to the list. I think adding this comment is the best we can do here.

**Comment by numismatika** [comment]
@J0WI agreed.

File: src/chrome/content/rules/Tagesanzeiger.ch.xml [hunk 1]
```diff
@@ -1,38 +1,318 @@
...
+		- longform.tagesanzeiger.ch
+		- marktplatz.tagesanzeiger.ch
+		- seminare.tagesanzeiger.ch
+		- tippinho.tagesanzeiger.ch
+		- webspecial.tagesanzeiger.ch
+		- editor.webspecial.tagesanzeiger.ch
+		- weiterbildung.tagesanzeiger.ch
 
-	Fully covered hosts:
+	Redirect to plain:
```
Comment (J0WI) [inline src/chrome/content/rules/Tagesanzeiger.ch.xml]: This section is now empty.

File: src/chrome/content/rules/Tagesanzeiger.ch.xml [hunk 2]
```diff
@@ -1,38 +1,318 @@
...
+	<target host="wwwt.tagesanzeiger.ch" />
+	<target host="wwww.tagesanzeiger.ch" />
+	<target host="wwwww.tagesanzeiger.ch" />
+	<target host="zalando.tagesanzeiger.ch" />
+	<target host="zomm.tagesanzeiger.ch" />
+	<target host="zoom.tagesanzeiger.ch" />
+	
+	
+	
+	<!-- TEST -->
```
Comment (J0WI) [inline src/chrome/content/rules/Tagesanzeiger.ch.xml]: Please clean this up

File: src/chrome/content/rules/Tagesanzeiger.ch.xml [hunk 3]
```diff
@@ -1,38 +1,318 @@
...
+	<target host="leer.tagesanzeiger.ch"/>
+	<target host="leichtathletik.tagesanzeiger.ch"/>
+	<target host="lichtbild.tagesanzeiger.ch"/>
+	<target host="lift.tagesanzeiger.ch"/>
+	<target host="listen.tagesanzeiger.ch"/>
+	<target host="m.tagesanzeiger.ch"/>
+	
+	
+	
+	<!-- TEST2 -->
```
Comment (J0WI) [inline src/chrome/content/rules/Tagesanzeiger.ch.xml]: Please clean this up

**Pull request merged**